packages/stryker-webpack-transpiler/src/fs/InputFileSystem.ts

```diff
@@ -0,0 +1,126 @@
+import MemoryFS from './MemoryFS';
+import { webpack, Callback } from '../types';
+import * as fs from 'fs';
+const errors = require('errno');
+import { dirname } from 'path';
+import { getLogger } from 'log4js';
+
+export default class InputFileSystem implements webpack.InputFileSystem {
```

Comment:
It would be nice to see `public` keywords defined explicitly.

Reply:
Meh... I don't mind either way. We don't have a strict rule for it either. I will add them here.